telethon/events/callbackquery.py

```diff
@@ -26,9 +26,13 @@ class CallbackQuery(EventBuilder):
             can use ``re.compile(b'data_')``.
     """
     def __init__(
-            self, chats=None, *, blacklist_chats=False, func=None, data=None):
+            self, chats=None, *, blacklist_chats=False, func=None, data=None, pattern=None):
         super().__init__(chats, blacklist_chats=blacklist_chats, func=func)
 
+        if data is None and pattern is None:
+            self._log.warn("Please don't use both data and pattern.data will be ignored if you do so")
```

Comment:
This should be a `ValueError('Use either data or pattern, not both')`.

Reply:
Shouldn't this work similarly to incoming and outgoing when they both get passed?

Reply:
Exactly:
https://github.com/LonamiWebs/Telethon/blob/4f1edeb7500a67858622c3407f33225754ac6c47/telethon/events/newmessage.py#L50-L52